x-pack/test_serverless/functional/test_suites/security/cypress/runner.ts

```diff
@@ -17,6 +18,10 @@ export async function SecuritySolutionCypressTestRunner(
   command: string
 ) {
   const log = getService('log');
+  const config = getService('config');
+  const esArchiver = getService('esArchiver');
+
+  await esArchiver.load('x-pack/test/security_solution_cypress/es_archives/auditbeat');
```

Comment:
Correct. In order to be able to use our application we need to have data on the instance, if not, a splash screen is going to be displayed telling you to ingest data first. Most of our tests relies on that data, we did it that way to avoid loading it per each test with the aim of improving the execution time of the tests.